controllers/secret.go

```diff
@@ -24,187 +24,108 @@ import (
 	certv1alpha1 "github.com/redhat-openshift-ecosystem/operator-certification-operator/api/v1alpha1"
 	corev1 "k8s.io/api/core/v1"
 	k8errors "k8s.io/apimachinery/pkg/api/errors"
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
 const (
-	KUBECONFIG_SECRET = "kubeconfig"
-	GITHUB_API_SECRET = "github-api-token"
-	GITHUB_TOKEN      = "GITHUB_TOKEN"
-	PYXIS_API_SECRET  = "pyxis-api-secret"
-	PYXIS_API_KEY     = "pyxis_api_key"
+	KUBECONFIG_SECRET_NAME     = "kubeconfig"
```

Comment:
Do we need to make all these values publicly available? I believe we should keep them private and follow go convention to start these vars with small letters.

Reply:
Good call, we only need them in the `controllers` package right now.

Reply:
yes you are right seems like camel case is the go-style will change from c-style